mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -288,6 +289,15 @@ class ALSModel private[ml] (
   @Since("2.2.0")
   def setColdStartStrategy(value: String): this.type = set(coldStartStrategy, value)
 
+  /**
+   * Set block size for stacking input data in matrices.
+   * Default is 4096.
```

Comment:
Oh, I think this is about to be a default of 1024, after the very latest PR from @zhengruifeng is merged. I think it's probably good to go so will merge it.

Reply:
https://github.com/apache/spark/pull/27374

Reply:
Thanks for the comment. I actually saw the default changed to 1024 in that PR, but I want the default to be 4096, that's why I set it explicitly in line 675 in the Estimator
```setDefault(blockSize -> 4096)```. 

I want the default to be 4096 because the ```blockify``` has 4096 as default. I don't want to change the current default value.
```
  private def blockify(
      factors: Dataset[(Int, Array[Float])],
      blockSize: Int = 4096): Dataset[Seq[(Int, Array[Float])]] = {
    import factors.sparkSession.implicits._
    factors.mapPartitions(_.grouped(blockSize))
  }
```

Reply:
OK, sounds fine.